html/pfappserver/lib/pfappserver/PacketFence/Controller/User.pm

```diff
@@ -367,6 +375,35 @@ sub create :Local :AdminRoleAny('USERS_CREATE') :AdminRoleAny('USERS_CREATE_MULI
         $c->stash->{form_import} = $form_import;
     }
 }
+=head2 _add_sms_source
+
+Add create SMS source if it is configured
```

Comment:
I'm not sure I understand the sense of this sentence

A rephrase would be useful